examples/helm/docker-registry-ui/values.yaml

```diff
@@ -15,7 +15,7 @@ ui:
   image:
     registry: docker.io
     repository: joxit/docker-registry-ui
-    tag: static
+    tag: latest
```

Comment:
For the same behaviour as old static tag, we need more changes. See https://github.com/Joxit/docker-registry-ui/wiki/Migrating-from-1.x-to-2.x will you be able to do it?

Reply:
If I'm correct I think only the parameter single_repository needs to be added. The registry.url already is set in REGISTRY_URL, so I don't think any updates there are needed.

As this was a breaking change, I guess old implementations with helm charts don't need to be kept working. I also have no clue of how it used to work, as I started using the latest version.

Reply:
The behavior of the parameter `REGISTRY_URL` has changed from v1 to v2. In v1 `REGISTRY_URL` acted as a proxy, in v2 `REGISTRY_URL` act as a XSS request so this will break all configurations with `ui.proxy = true`. As long as it is possible to avoid breaking changes, I prefer to do it :)

Okay, so in this file, rename `REGISTRY_URL` to `NGINX_PROXY_PASS_URL` and `URL` to `REGISTRY_URL` 
https://github.com/Joxit/docker-registry-ui/blob/7eed05ae50b90c485e709101a170f493e6f78cf9/examples/helm/docker-registry-ui/templates/_helpers.tpl#L108-L114

And yes you must add `SINGLE_REGISTRY = true`